src/pattern-library/components/Library.tsx

```diff
@@ -10,33 +10,10 @@ import { jsxToHTML } from '../util/jsx-to-string';
 /**
  * Components for rendering component documentation, examples and demos in the
  * pattern-library page.
- *
- * Example of structure:
- *
- * <Library.Page intro={<p>Some introductory content</p>} title="Elephants">
- *   <p>Any content you want on the page.</p>
- *
- *   <Library.Section title="ComponentName">
- *     <Library.Pattern>
- *       <p>The `Elephant` component is used to render information about elephant
- *       personalities.</p>
- *       <Library.Example title="Colored elephants">
- *         <p>You can change the color of your elephant.</p>
- *         <Library.Demo withSource>
- *           <Elephant color="pink" />
- *         </Library.Demo>
- *         // More Demos if desired...
- *       </Library.Example>
- *       // More Examples if desired...
- *     </Library.Pattern>
- *   // more Patterns if desired...
- *   </Library.Section>
- *
- *   // More Sections...
- *
- * </Library.Page>
```

Comment:
Now that it could be as simple as

```
<Page>
  <Section>
    <Section />
  </Section>
</Page>
```

I'm removing this long-winded example. That doesn't mean that we have to kill all documentation on this, I'm just not sure what would be most useful.